genie-web/src/main/java/com/netflix/genie/web/tasks/job/JobCompletionService.java

```diff
@@ -154,6 +156,19 @@ void handleJobCompletion(final JobFinishedEvent event) throws GenieException {
         final Map<String, String> tags = MetricsUtils.newSuccessTagsMap();
 
         try {
+            /*
+             * Get the cluster and command chosen for the job for tagging metrics.
+             * Swallowing any exception in getting those as this step is not
+             * critical and should not show that the entire job failed
+             */
+            try {
+                final Cluster cluster = jobSearchService.getJobCluster(jobId);
+                final Command command = jobSearchService.getJobCommand(jobId);
+                MetricsUtils.addCommonJobWorkflowMetricTags(cluster, command, tags);
```

Comment:
Calling this method from outside the Job Workflow now doesn't make sense for this name...

Reply:
In that case I will rename the method to something like addResourceTags. Does that make sense @tgianos ?